airgun/entities/host.py

```diff
@@ -160,6 +160,8 @@ def schedule_remote_job(self, entities_list, values, timeout=60, wait_for_result
 
     def play_ansible_roles(self, entities_list, timeout=60, wait_for_results=True):
         """Play Ansible Roles on hosts names in entities_list
+           If keyword 'All' is supplied instead of list, all hosts are selected
+           using the checkbox from table header
```

Comment:
Is this statement still valid? I don't see conditional statement for 'All' in this play_ansible_roles method

Reply:
Support is in `HostsSelectAction.step()`, which does most of the work for many of these actions. Note `self._select_action('Play Ansible roles', entities_list)` in method body.

Reply:
okay make sense